src/actions/guides/database/validating-saving-deleting.cr

```diff
@@ -8,676 +8,64 @@ class Guides::Database::ValidatingSavingDeleting < GuideAction
 
   def markdown
     <<-MD
-    ## Setting up a form
+    ## Form Objects
 
-    When you set up a model, a `{ModelName}::BaseForm` will be created so that you can inherit
-    from it and customize validations, callbacks, and what fields are allowed to be
-    filled. `{ModelName}::BaseForm` automatically defines a form field for each model field.
+    How Lucky uses these, the structure, and where they go in the app. 
 
-    We’ll be using the migration and model from the [Querying
-    guide](#{Guides::Database::Querying.path}). Once you have that set up, let’s set
-    up a form:
+    ## Fillable Fields
 
-    ```crystal
-    # src/forms/user_form.cr
-    class UserForm < User::BaseForm
-    end
-    ```
+    ## Creating Records
 
-    > You can have more than one form object to save a record in different ways.
-    For example, default Lucky apps have a `SignUpForm` specifically for handling
-    user sign ups.
+    Examples of passing in `params`, or how it would look being used in a task (i.e. seeing).
```

Comment:
Yeah and I like the idea of showing how to do it without params first, and then with them later. That way it is clear you pass in named args

Reply:
Yeah, I'd like to show an example of what the params look like. So you know when you pass in `params`, it looks like `{"user:name" => "whatever"}` or however that actually looks. I had to boot an app, make a form, and then inspect just to figure out what that looks like 😅

Reply:
The params are a bit tricky though because you probably should never pass them in yourself in that way. It's almost exclusively for HTML form params/JSON params. If you need to do something not user generated it should pretty much always use the named args for type safety. Maybe that is good to include too?

Like an example showing parsing a simple CSV and then using that in a form:

```crystal
# Total pseudo code. No idea if this is even in the std lib
csv_row = Csv.parse("Paul, Smith").first # Get the first row only
UserForm.create!(first_name: csv_row.first, last_name: csv_row.last)
```